ff/serving/client/examples/cli.py

```diff
@@ -0,0 +1,55 @@
+# Run in command line with:
+# python3 -m featureform examples/cli.py -host <hostname:port>
+from featureform import register as ff
+
+user = ff.register_user("test")
+user.make_default_owner()
+snowflake = ff.register_snowflake(
+    name="snowflake",
+    username="username",
+    password="password",
+    account="account",
+    organization="organization",
+    database="database",
+    schema="schema",
+    description="Snowflake",
+    team="Featureform Success Team",
```

Comment:
no team, organization, atm.

Reply:
Created an asana task to add these into the snowflake config

Reply:
ok, but should they also be in metadata for the providers?

Reply:
Actually org is already being used. Its mainly just needed for auth. We can add Team in later. 

Org is just a generated string, nothing we really need for anything else